grails-app/controllers/au/org/ala/volunteer/ForumController.groovy

```diff
@@ -366,7 +366,8 @@ class ForumController {
         def errors = []
         if ((message && !StringUtils.isEmpty(text)) && currentUser) {
             if (!forumService.isMessageEditable(message, currentUser)) {
-                throw new RuntimeException("You do not have sufficient privileges to edit this message!")
+                //throw new RuntimeException("You do not have sufficient privileges to edit this message!")
+                errors << "You do not have sufficient privileges to edit this message. Please contact DigiVol admin if you think this is an error."
```

Comment:
Should you be sending a redirect and returning here?  It looks like a save happens later in the method.